roles/openstack_rabbitmq/tasks/main.yml

```diff
@@ -11,8 +11,13 @@
     stockpile_openstack_rabbitmq_file_descriptors: "{{ rabbitmq_desc.stdout }}"
   when: rabbitmq_desc is succeeded and containers is not defined and rabbitmq_desc.stdout != ""
 
+- name: get rabbitmq bundle name - container
+  shell: "{{ container_cli }} ps | grep rabbit | awk '{print $(NF)}'"
+  become: true
+  register: rabbitmq_name
```

Comment:
Missing ignore_errors. This task will fail when there is no rabbitmq container, for example on a compute node.

Reply:
I ran the playbook http://paste.openstack.org/show/788397/

Reply:
That is because of the awk, the grep alone would fail. Anyway, best practice is to ignore_errors for things tasks that have shell.

Reply:
Done

Reply:
Not big on awk, but iirc this would make most sense if we're confident that there'll be only one rabbitmq container, however this would cause issues if there's more than one, so maybe we can add a simple check for it? I'm not 100% sure if there'll always be a single rabbitmq container running, but moving forward this will act as a sanity check. IIRC nova had multiple containers

Reply:
Unresolving this, as i don't think my concerns have been addressed ?

Reply:
@aakarshg I tested it manually on osp13 and osp16  and i could see only one container on each controller node http://paste.openstack.org/show/788468/